tests/resources/wallet/test_thin_wallet.py

```diff
@@ -65,16 +65,24 @@ def test_post(self):
         tx = Transaction(inputs=[i], outputs=[o])
 
         data_to_sign = tx.get_sighash_all()
+        self.log.debug('XXXXX', x=1)
```

Comment:
Should we remove the debug logs?

Reply:
Oh! Good catch, I forgot it from a test I was doing. Will remove.

Reply:
Removed.